server/service/devices.go

```diff
@@ -167,11 +162,36 @@ func getDeviceHostEndpoint(ctx context.Context, request interface{}, svc fleet.S
 		resp.DEPAssignedToFleet = ptr.Bool(hdep.IsDEPAssignedToFleet())
 	}
 
+	softwareInventoryEnabled := ac.Features.EnableSoftwareInventory
+	if resp.TeamID != nil {
+		// load the team to get the device's team's software inventory config.
+		tm, err := svc.GetTeam(ctx, *resp.TeamID)
+		if err != nil && !fleet.IsNotFound(err) {
+			return getDeviceHostResponse{Err: err}, nil
+		}
+		if tm != nil {
+			softwareInventoryEnabled = tm.Config.Features.EnableSoftwareInventory
+		}
+	}
+
+	deviceGlobalConfig := fleet.DeviceGlobalConfig{
+		MDM: fleet.DeviceGlobalMDMConfig{
+			// TODO(mna): is this used at all on the frontend? It currently only
+			// returns the Apple enabled and configured, regardless of the platform
+			// of the device.
+			EnabledAndConfigured: ac.MDM.EnabledAndConfigured,
```

Comment:
I think this is because windows doesn't require any user action, whereas we have a few macOS features that display banners.

In any case the naming is extremely misleading

Reply:
Gotcha, yeah that was unrelated to the change, just thought it was weird and wanted to clarify. I'll leave the TODO there (I'll add a link to this comment in the other PR) so if/when we change something related to this, we can address it.